lib/cargoQueue.js

```diff
@@ -0,0 +1,109 @@
+import queue from './internal/queue';
+
+/**
+ * A cargoQueue of tasks for the worker function to complete. CargoQueue inherits all of
+ * the same methods and event callbacks as [`queue`]{@link module:ControlFlow.queue}.
+ * @typedef {Object} CargoObject
```

Comment:
You don't need to redefine the CargoObject, you can refer to the one defined in cargo.js

Reply:
Should I rename it  here to be `@typedef {Object} CargoQueueObject` ?

Reply:
The returned object has the same shape as the CargoObject, so there's no need for a duplicate JSDoc definition.